platform_secrets/service.py

```diff
@@ -1,28 +1,97 @@
 from __future__ import annotations
 
+import hashlib
 import logging
+import math
 import re
 from dataclasses import dataclass, field
-from typing import Optional
+from typing import Any, Optional
 
-from .kube_client import (
-    KubeClient,
+from apolo_kube_client.errors import (
     ResourceBadRequest,
-    ResourceConflict,
+    ResourceExists,
     ResourceInvalid,
     ResourceNotFound,
 )
 
+from .kube_client import KubeApi
+
 logger = logging.getLogger()
 
 SECRET_API_ORG_LABEL = "platform.neuromation.io/secret-api-org-name"
 APPS_SECRET_NAME = "apps-secrets"
 
 NO_ORG = "NO_ORG"
+NO_ORG_NORMALIZED = "no-org"
 
 NAMESPACE_ORG_LABEL = "platform.apolo.us/org"
 NAMESPACE_PROJECT_LABEL = "platform.apolo.us/project"
 
+KUBE_NAME_LENGTH_MAX = 63
+KUBE_NAMESPACE_SEP = "--"
+KUBE_NAMESPACE_PREFIX = "platform"
+KUBE_NAMESPACE_HASH_LENGTH = 24
+
+
+def generate_namespace_name(org_name: str, project_name: str) -> str:
```

Comment:
may be we can move this function to apolo kube client and reuse it everywhere. wdyt?
also let's add unit tests for it

Reply:
@zubenkoivan I think you are right. IMO it shouldn't be there, but it's a better option to have it there, rather than duplicate the code. I'll move it and put a todo to remove it later from the client.
There is also a separate ticket to define a common network policy for namespace, so considering this as well - I agree it should be moved to a cli, at least temporarily. 
Will work on this